### PR TITLE
BNH-65 Send emails to a list of admins

### PR DESCRIPTION
**Pull request by Chestnut-lol**

### CODE REVIEW

File: web/Services/MailService.cs [hunk 1]
```diff
@@ -31,26 +31,27 @@ public MailService(IOptions<EmailConfigOptions> config)
         string msgToName = ""
```
Comment (BenRamchandani) [inline web/Services/MailService.cs:31]: We should probably make message body subject required arguments, and put a sensible value in for the subject. Can be another PR though

**Review by BenRamchandani — APPROVED** [review]
Looks good. Don't forget to update appsettings.Development.json in Keeper when you merge 🙂

File: web/Services/MailService.cs (hunk 1 above)
Comment (Chestnut-lol) [reply web/Services/MailService.cs:31]: Can I add it onto another ticket that I'm working on? (the one that's dealing with sending email exceptions)

Comment (BenRamchandani) [reply web/Services/MailService.cs:31]: Sure
